public/app/core/components/NavBar/Next/NavBarItemWithoutMenu.tsx

```diff
@@ -84,7 +84,8 @@ export function getNavBarItemWithoutMenuStyles(theme: GrafanaTheme2, isActive?:
       color: 'inherit',
       display: 'block',
       padding: 0,
-      textAlign: 'center',
+      overflowWrap: 'anywhere',
+      whiteSpace: 'normal',
```

Comment:
Rather than 'unsetting' all these values back, can we just remove the white-space: nowrap that's on the root nav component?

Reply:
@joshhunt we sure can, thanks for spotting 🚀